kolibri/deployment/default/settings/base.py

```diff
@@ -138,7 +138,7 @@
     os.makedirs(CONTENT_STORAGE_DIR)
 
 # Base default URL for downloading content from an online server
-CENTRAL_CONTENT_DOWNLOAD_BASE_URL = "http://contentworkshop.learningequality.org"
+CENTRAL_CONTENT_DOWNLOAD_BASE_URL = "https://studio.learningequality.org"
```

Comment:
We force https anyway, so better to just use https here to avoid an additional redirect.